src/ProducerEventProcessor/include/ProducerEventProcessor/ProducerEventProcessor.h

```diff
@@ -7,6 +7,8 @@
 
 #include <stdint.h>
 
+#include <optional>
+
```

Comment:
This is not needed anymore. (Came from the members that were now moved to the Impl-class.)

Reply:
Done.